draft-irtf-nwcrg-coding-and-congestion-04.xml

```diff
@@ -441,13 +441,13 @@
 				<t>The research questions should be mapped following the oranization of this document. In all these three use-cases, a open questions remains and is inherent to the usage of coding. There is a trade-off between reducing goodput, helping to recover from /
 					and congestion losses, being fair in respect with non-coded flows and avoiding transmitting useless repaired symbols.</t>
 				<t>For the FEC above transport case, there is a trade-off between the amount of redundancy to add, as a function of the transport layer protocol and application requirements.</t>
-				<t>For the FEC within transport case, the addition of coding may hide congestion losses and proposing methods to signal such events to end point could be propose.</t>
+				<t>For the FEC within transport case, recovering lost symbols may hide congestion losses to the congestion control. Some existing solutions already propose to disambiguate acked packets from rebuilt packets <xref target="QUIC-FEC"></xref>. New signalling methods and FEC-recovery-aware congestion controls could be proposed.</t>
```

Comment:
My PR also changes this paragraph; I prefer this one over mine. A job for the editor to take this version and not the other  :)